source/tomster/commission.html.erb

```diff
@@ -65,26 +65,38 @@ title: "Tomster Commission Request"
               <span class="choice">
                 <input id="Field8_0" name="Field8" type="radio" class="field radio" value="Classic Tomster with Simple Shirt Change, $80" tabindex="5" checked="checked" required>
                 <label for="Field8_0">
-                  Classic Tomster or Zoey with Simple Shirt Change, $80
+                  UG Classic Tomster or Zoey with Simple Shirt Change, $80
```

Comment:
UG standing for user group I'm assuming?  Think it'd be worth spelling that out so it's clearer on this form?